src/linagora.esn.james/app/quota/domain/james-quota-domain.controller.spec.js

```diff
@@ -44,7 +44,7 @@ describe('The JamesQuotaDomainController', function() {
   }
 
   describe('The getDomainQuota function', function() {
-    it('should set the status to loading while attempting to get domain quota ', function() {
+    it('should set the status to loading while attempting to get domain quota', function() {
```

Comment:
The commit message for this is "#61: Fixed '$q is undefined' error in beforeEach blocks" ???

Reply:
F*ck, there was a fixup to remove some changes in that commit. I need to remove the commit also. Just a sec.

Reply:
Done.